sdk/keyvault/azure-security-keyvault-certificates/src/test/java/com/azure/security/keyvault/certificates/CertificateClientTest.java

```diff
@@ -364,6 +347,15 @@ public void cancelCertificateOperation(HttpClient httpClient, CertificateService
         });
     }
 
+    @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
+    @MethodSource("getTestParameters")
+    public void cancelCertificateCreationOperation(HttpClient httpClient, CertificateServiceVersion serviceVersion) {
```

Comment:
We could probably rename this to `createCertificateOperation` and have the method above it be called `createCertificateOperationThroughPoller` or something like that, feels more descriptive of what both do.